test/python/circuit/test_parameters.py

```diff
@@ -1018,6 +1021,40 @@ def test_binding_parameterized_circuits_built_in_multiproc(self):
             )
         )
 
+    def test_binding_parameterized_circuits_built_in_multiproc(self):
+        """Verify subcircuits built in a subprocess can still be bound.
+        REMOVE this test once assemble is REMOVED"""
```

Comment:
I think this was a copy-pasting error. This is the new test that won't be removed.
```suggestion
        """Verify subcircuits built in a subprocess can still be bound."""
```